tests/integration/test_install.py

```diff
@@ -0,0 +1,88 @@
+# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Test cylc install."""
+
+from pathlib import Path
+from shutil import rmtree
+from tempfile import TemporaryDirectory
+
+import pytest
+
+from .test_scan import init_flows
+
+from cylc.flow.workflow_files import WorkflowFiles
+from cylc.flow.scripts.install import (
+    InstallOptions,
+    install_cli
+)
+
+
+SRV_DIR = Path(WorkflowFiles.Service.DIRNAME)
+CONTACT = Path(WorkflowFiles.Service.CONTACT)
+RUN_N = Path(WorkflowFiles.RUN_N)
+INSTALL = Path(WorkflowFiles.Install.DIRNAME)
+
+
+@pytest.fixture()
+def src_run_dirs(mock_glbl_cfg, monkeypatch):
```

Comment:
I think you can use the `tmp_path` fixture to simplify this a bit
```suggestion
def src_run_dirs(mock_glbl_cfg, monkeypatch, tmp_path: Path):
```